metadata.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "saz/ssh",
-      "version_requirement": ">= 2.9.1 < 3.0.0"
+      "version_requirement": ">= 2.9.1 < 5.0.0"
```

Comment:
Could you amend the commit message and PR title?  I think it should be `allow saz/ssh 4` not `3`